assets/js/components/conversion-tracking/ConversionTrackingToggle.js

```diff
@@ -0,0 +1,145 @@
+/**
+ * Site Kit by Google, Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import {
+	createInterpolateElement,
+	useState,
+	Fragment,
+} from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { Switch } from 'googlesitekit-components';
+import Data from 'googlesitekit-data';
+import { CORE_SITE } from '../../googlesitekit/datastore/site/constants';
+import ErrorText from '../../components/ErrorText';
+import Link from '../Link';
+import LoadingWrapper from '../LoadingWrapper';
+import ConfirmDisableConversionTrackingDialog from './ConfirmDisableConversionTrackingDialog';
+import { useFeature } from '../../hooks/useFeature';
+import PropTypes from 'prop-types';
+
+const { useDispatch, useSelect } = Data;
+
+export default function ConversionTrackingToggle( { loading } ) {
+	const iceEnabled = useFeature( 'conversionInfra' );
+	const [ saveError ] = useState( null );
+	const [ showConfirmDialog, setShowConfirmDialog ] = useState( false );
+
+	const isConversionTrackingEnabled = useSelect( ( select ) =>
+		select( CORE_SITE ).isConversionTrackingEnabled()
+	);
+
+	const isSaving = useSelect( ( select ) =>
+		select( CORE_SITE ).isFetchingSaveConversionTrackingSettings()
+	);
+
+	const { setConversionTrackingEnabled } = useDispatch( CORE_SITE );
+
+	if ( ! iceEnabled ) {
+		return null;
+	}
+
+	return (
+		<Fragment>
+			<div>
+				{
```

Comment:
Looks like this might erroneous; maybe there was originally a condition here? 🤔

Reply:
Good spot, yes I had a condition prior.

Reply:
Solved in latest set of commits @tofumatt